src/sql/workbench/browser/modelComponents/card.component.ts

```diff
@@ -236,3 +276,23 @@ export default class CardComponent extends ComponentWithIconBase<azdata.CardProp
 
 	}
 }
+
+registerThemingParticipant((theme: IColorTheme, collector: ICssStyleCollector) => {
+	const backgroundColor = theme.getColor(SIDE_BAR_BACKGROUND);
```

Comment:
Why are you re-using SIDE_BAR colors? It'd be better to make new colors (and have them default to the same as the side bar if you want) so users can actually control them separately.

Reply:
Added two new colors in the theme for the card that default to the side bar colors.